### PR TITLE
Completed Typescript Operator Expressions - Carolyne Golias

### PR DESCRIPTION
**Pull request by GCarolyne**
Screenshot 2024-12-09 at 6 31 08 PM

### CODE REVIEW

File: typescript-operators-and-expressions/main.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+console.log('Is the ph acidic', isAcidic);
+console.log('What is the type', isAcidic);
+
+const headCount: number = 220;
+const isSparta: boolean = headCount === 300;
+console.log('What is it Sparta', isSparta);
+console.log('The typeof', typeof isSparta);
+
+let motto: string = fullName;
+motto = fullName + ' is the Goat';
```
Comment (shawnkost) [inline typescript-operators-and-expressions/main.ts]: This should be done using the `+=` operator

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change here Carolyne!

**Comment by GCarolyne** [comment]
Screenshot 2024-12-10 at 8 48 16 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**